interface/ten_ai_base/tts2.py

```
@@ -403,19 +402,19 @@ def metrics_reset(self) -> None:
         self.recv_audio_duration = 0
         self.recv_audio_chunks_len = 0
 
-    async def metrics_connect_delay(self, connect_delay_ms: int):
-        data = Data.create("metrics")
+    async def metrics_connect_delay(self, connect_delay_ms: int, extra_metadata: dict = None):
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)